lib/rules/new-line-before-expect.js

```diff
@@ -12,6 +12,8 @@ var blockRegexp = /^((f|x)?(it|describe))$/
 module.exports = function (context) {
   var suiteDepth = 0
   var lastExpectNode
+  var lineWithWhitespace
+  var sourceCode = context.getSourceCode()
```

Comment:
Can we move 2 new vars closer to where it is being used. It might look like we declare all the vars on top, but this wasn't the intention. The suiteDepth, lastExpectNode needed to be declared outside of the return statement.

Reply:
sure thing.